geode-core/src/integrationTest/java/org/apache/geode/internal/cache/OplogRVVJUnitTest.java

```diff
@@ -75,6 +75,7 @@ public void testRecoverRVV() {
     final DiskStoreID m1 = DiskStoreID.random();
     final DiskStoreID m2 = DiskStoreID.random();
     final DiskRecoveryStore drs = mock(DiskRecoveryStore.class);
+    when(parent.getWriteBufferSize()).thenReturn(32768);
```

Comment:
Could hard-coded uses of this value (when it relates to Oplog buffer sizes) be replaced with references to `DiskStoreFactory.DEFAULT_WRITE_BUFFER_SIZE`?